src/Kafka/Settings/Settings.stories.tsx

```diff
@@ -0,0 +1,86 @@
+import type { AlertProps } from "@rhoas/app-services-ui-shared";
+import { AlertVariant } from "@rhoas/app-services-ui-shared";
+import type { ComponentStory, ComponentMeta } from "@storybook/react";
+import { useState } from "react";
+import { Settings } from "./Settings";
+import { fakeApi } from "../../shared/storiesHelpers";
+
+export default {
+  component: Settings,
+  args: {},
+} as ComponentMeta<typeof Settings>;
+
+const Template: ComponentStory<typeof Settings> = () => {
+  const [isChecked, setIschecked] = useState<boolean>(true);
+
+  const [isModalOpen, setIsModalOpen] = useState<boolean>(false);
+
+  const [showAlert, setShowAlert] = useState<AlertProps[]>([]);
+
+  const [isLoading, setIsLoading] = useState<boolean>(false);
+
+  const onChange = () => {
+    setIsModalOpen(true);
+  };
+
+  const addAlert = (
+    title: string,
+    variant: AlertProps["variant"],
+    description: string,
+    id: string | undefined
+  ) => {
+    setShowAlert([...showAlert, { title, variant, description, id }]);
+  };
+
+  const onDisable = () => {
+    setIsModalOpen(false);
+    setIschecked(false);
+    setIsLoading(true);
+    fakeApi<{ ConfigurationOff: boolean }>(
+      {
+        ConfigurationOff: false,
+      },
+      2000
+    )
+      .then(() => {
+        addAlert(
+          "Connection re-authentication is turrning off",
+          AlertVariant.success,
+          "Broker is restarting to apply the new configurations. This process might take several minutes",
```

Comment:
A few notes about this:
1. These strings should be moved to the translation file. 
2. A success should display when changing the value of this setting to either `off` or `on`, and not just when changing the value to `off`.
3. The message should only display when the API response to changing the value is successful
4. The message should not provide important details about how long it takes for settings to fully take affect.

Following the points above, there are some text string changes needed in the toast alert:

**Re: points 2. & 3.**, the alert title should change to one of the following depending on the value change:

- If turning off the setting:
    > Connection re-authentication turned off
- If turning on the setting: 
    > Connection re-authentication turned on

**Re: point 4.**, the message body text should be removed (i.e. "Broker is restarting...")